docs/WAF/access-control.md

```diff
@@ -0,0 +1,65 @@
+# IP Access Control
+
+The **IP Access Control** feature allows administrators to manage network access and rate limiting at the IP address level. This feature complements the default Web Application Firewall (WAF) protections by enabling more granular control over which IPs are blocked or allowed to bypass rate limits.
+
+For more information on WAF and default rate limits, see the [Firewall documentation](https://docs.altis-dxp.com/cloud/firewall/).
+
+## Feature Overview
+
+- **Blocklist**: Deny all access from specific IP addresses or ranges.
+- **Allowlist**: Permit specified IP addresses or ranges to bypass WAF rate limits.
+
+This feature is accessible from the Altis Dashboard and is designed for simple, effective access management.
+
+## Blocklist
+
+The **Blocklist** allows you to deny all traffic originating from specified IP addresses or IP ranges. Any request from a blocklisted IP will be automatically rejected and will not reach your application.
+
+**Use Cases:**
+- Blocking malicious actors or IPs with suspicious activity.
+- Preventing access from certain data centers usually used for spam traffic. 
```

Comment:
"usually" -> "often" 
I don't think any data centre is set up to usually support spammers.